frontend/javascripts/test/puppeteer/dataset_rendering_helpers.ts

```diff
@@ -357,12 +357,13 @@ export function setupBeforeEachAndAfterEach() {
   test.beforeEach(async (t) => {
     t.context.browser = await puppeteer.launch({
       args: [
-        "--headless",
+        "--headless=new",
         "--hide-scrollbars",
         "--no-sandbox",
         "--disable-setuid-sandbox",
         "--disable-dev-shm-usage",
-        "--use-gl=swiftshader",
+        // "--use-gl=swiftshader",
+        // "--enable-gpu",
       ],
       dumpio: true,
     });
```

Comment:
AS this also ready like it is ignored code, I'd say we can comment it out for now and check whether the CI still works. If it does, I'd say we remove this completely

Reply:
I uncommented this code and added a comment explaining why. We should discuss this at some point with @philippotto and/or @daniel-wer . See my conversation/comment above.

Reply:
Success